protocol/flows/src/v5/ibd/flow.rs

```diff
@@ -294,6 +294,7 @@ impl IbdFlow {
         let mut trusted_set = pkg.build_trusted_subdag(entries)?;
 
         if self.ctx.config.enable_sanity_checks {
+            let con = self.ctx.consensus().unguarded_session_blocking();
```

Comment:
Since `unguarded_session_blocking()` now exists here, should that be used in the sync and validate way above for this line as well?

```
let proof = consensus.clone().spawn_blocking(move |c| c.validate_pruning_proof(&proof).map(|()| proof)).await?;
```

Reply:
No. The difference is that here we are already going into a blocking fn call for staging consensus, so we can access the current consensus via the blocking interface as well.